src/js/utils/DOMUtils.js

```diff
@@ -28,21 +29,19 @@ var DOMUtils = {
 
     var computeInnerBound = function (acc, key) {
       var val = parseInt(compstyle[key], 10);
-      if (_.isNaN(val)) {
+      if (Number.isNaN(val)) {
```

Comment:
Right, but the discussion started with `isNaN` and `_.isNaN` not being the same thing. So I wanted to make sure if we use `NaN` that it is covering the cases that we need.